src/cpp/clusterinfocommand.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ * BSD 2-Clause License
+ *
+ * Copyright (c) 2021, Hewlett Packard Enterprise
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include "clusterinfocommand.h"
+#include "redisserver.h"
+
+using namespace SmartRedis;
+
+parsed_reply_map ClusterInfoCommand::parse_db_cluster_info(std::string info)
+{
+    parsed_reply_map info_map;
+
+    std::string delim = "\r\n";
```

Comment:
This is risky -- end-of-line varies depending on whether you're on UNIX (\r), Windows (\r\n), or Mac (\n). If we can run the server on different platforms, can we get different EOL sequences?

Reply:
The INFO [documentation](https://redis.io/commands/info) says "All the properties are in the form of field:value terminated by \r\n." Do you think that depends on the platform? I thought that implied its uniform across all platforms.

Reply:
ok, cool. I still don't trust it completely, but it's good that it's separated out in a constant that we can generalize the code with if they ever change it out from under us